src/pages/ReimbursementAccount/BeneficialOwnersStep.js

```diff
@@ -44,11 +46,11 @@ class BeneficialOwnersStep extends React.Component {
         this.submit = this.submit.bind(this);
 
         this.state = {
-            ownsMoreThan25Percent: false,
-            hasOtherBeneficialOwners: false,
-            acceptTermsAndConditions: false,
-            certifyTrueInformation: false,
-            beneficialOwners: [],
+            ownsMoreThan25Percent: lodashGet(props, ['reimbursementAccountDraft', 'ownsMoreThan25Percent'], false),
+            hasOtherBeneficialOwners: lodashGet(props, ['reimbursementAccountDraft', 'hasOtherBeneficialOwners'], false),
+            acceptTermsAndConditions: lodashGet(props, ['reimbursementAccountDraft', 'acceptTermsAndConditions'], false),
+            certifyTrueInformation: lodashGet(props, ['reimbursementAccountDraft', 'certifyTrueInformation'], false),
+            beneficialOwners: lodashGet(props, ['reimbursementAccountDraft', 'beneficialOwners'], []),
```

Comment:
NAB, but feels like this should also be using the `getDefaultStateForField()` method? If these aren't stored in `achData` that's fine right? They'll either use the `reimbursementAccountDraft` or return the default?

Reply:
Mostly I'm thinking ahead and that at some point we'll want to refactor all this - it will be good to have this pattern applied consistently.

Reply:
> If these aren't stored in achData that's fine right? 

These aren't stored in `achData` but as you mentioned, it would be fine to use `getDefaultStateForField()`. I don't have a strong preference here so I'll update these and the ones in `ValidationStep` as well.